### PR TITLE
Metadata updates

### PR DESCRIPTION
**Pull request by Phil-Friderici**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      "name": "pe",
+      "version_requirement": ">= 3.0.0 < 2015.3.0"
+    },
+    {
+      "name": "puppet",
+      "version_requirement": ">= 3.0.0 < 5.0.0"
+    }
+  ],
+  "dependencies": [
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 6.0.0"}
```
Comment (propyless) [inline metadata.json]: puppet-vsftpd does not require stdlib module.

Comment (propyless) [inline metadata.json]: puppet-vsftpd does not require stdlib module.

**Comment by Phil-Friderici** [comment]
soon it will

**Comment by Phil-Friderici** [comment]
removed stdlib to make the commit consistent

**Pull request merged**